manual/animation/scene-animations/tutorials/animate-ui.md

```diff
@@ -36,6 +36,6 @@ Property track editing depends on the value type. For colors and vectors, you ca
 
 ### 5. Play the animation
 
-The final step is to add Scene Animation Player actor and play this animation.
+The final step is to add Scene Animation Player actor to the scene. After adding the Animation Player to the scene you can then select the Scene Animation player in the Animation Player window. Next run the game in Flax Editor by selecting play, and then hit play on the Animation Player window to preview your animation.
```

Comment:
That's some pretty useful information! Though, is there any way to rephrase this to not repeat "Animation Player" so often?